src/wwwroot/app/directives/phones/phone-inputs.js

```diff
@@ -45,4 +48,24 @@ function PhoneInputsController($scope) {
          vm.phones.slice(index, 1);
       }
    }
+   function _removePhone(phoneToRemove) {
+      if (phoneToRemove.index) {
+         vm.phones = filter(vm.phones, phone => phone.index !== phoneToRemove.index);
+      } else {
+         vm.phones = filter(vm.phones, phone => phone.number !== phoneToRemove.number);
+      }
+      if (vm.phones.length === 0) {
+         _addNewPhone();
+      }
+   }
+   function _resolveSignOf(index) {
+      if (vm.phones.length === 1) {
+         return 'plus';
+      }
+      if (index === vm.phones.length) {
+         return 'plus';
+      } else {
+         return 'minus';
+      }
```

Comment:
you can write something like that:
(vm.phones.length === 1) || (index === vm.phones.length)  ?
 'plus' :
 'minus';